jekyll/_cci2/github-apps-integration.adoc

```diff
@@ -227,12 +219,12 @@ Deploy keys and user keys are not used by GitHub App integrations. All keys are
 [#test-insights]
 === Test Insights
 
-xref:insights-tests#[Test Insights] is currently not supported.
+xref:insights-tests#[Test Insights] is currently not supported for GitHub App integrations.
 
 [#only-build-pull-requests]
 === Only build pull requests
 
-The Only Build Pull Requests option (usually available in menu:Project Settings[Advanced] or within trigger setup options) is not currently supported for GitHub App integrations
+The Only Build Pull Requests option (usually available in menu:Project Settings[Advanced] or within trigger setup options) is not currently supported for GitHub App integrations.
```

Comment:
I think this is determined both by the org AND by the pipeline. Basically if you're in a standalone org you won't even see the option in setting. And if you're in a hybrid org and run GitHub App pipelines, the option won't work for them. 

I think we can say "is currently only available to organizations that authenticate through GitHub OAuth, and only applies to OAuth pipelines" or something like this?